pkg/commands/compute/compute_deploy_test.go

```diff
@@ -378,6 +378,62 @@ func TestDeploy(t *testing.T) {
 				"Deployed package (service 123, version 4)",
 			},
 		},
+		{
+			name: "success with --backend and no --backend-port",
+			args: args("compute deploy --token 123 --backend host.com"),
+			api: mock.API{
+				CreateServiceFn:   createServiceOK,
+				GetPackageFn:      getPackageOk,
+				UpdatePackageFn:   updatePackageOk,
+				CreateDomainFn:    createDomainOK,
+				CreateBackendFn:   createBackendExpect("host.com", 80, "", ""),
+				ActivateVersionFn: activateVersionOk,
+				ListDomainsFn:     listDomainsOk,
+			},
+			manifest: "name = \"package\"\n",
```

Comment:
NIT: I'd prefer to use back ticks so we don't have to escape the double quotes (repeat this suggestion throughout).

```suggestion
			manifest: `name = "package"\n`,
```

Reply:
Done.